### ISSUE TITLE
Wellknownlocations issue in digital signature

### ISSUE THREAD

**Issue by sanjuleo** [oleg-shilo/wixsharp#355]
Hi Oleg
My code signing is failing as signtool is not found, when I investigated the code, there seems to be a typo in the code -
 WellKnownLocations = wellKnownLocations ?? @"C:\Program Files\Microsoft SDKs\Windows\v6.0A\bin;" +
                                                           @"C:\Program Files (x86)\Microsoft SDKs\Windows\v7.1A\Bin;" +
                                                           @"C:\Program Files (x86)\Microsoft SDKs\ClickOnce\SignTool;" +
                                                           @"C:\Program Files(x86)\Windows Kits\8.0\bin\x86;" +
                                                           @"C:\Program Files(x86)\Windows Kits\8.1\bin\x86;" +
                                                           @"C:\Program Files(x86)\Windows Kits\10\bin\x86;" +
@"C:\Program Files(x86)\Windows Kits\10\bin\10.0.15063.0\x86",

Last 4 lines need a space after "Files" -> C:\Program Files (x86), as they are not pointing to a valid location.

Please update.

Regards

**Comment by oleg-shilo** [comment]
Thank you. Yes it is a mistake.
Corrected (3a3a3df6d74cf552fc60a329b0321c6ffa50e434).

You can use this work around until the release with the fix is available:

```C#
var original_path = Environment.GetEnvironmentVariable("PATH");
Environment.SetEnvironmentVariable("PATH", original_path+$";{<actual signtool location>}");
```

**Comment by sanjuleo** [comment]
Thanks.
